modules/ebi-metagenomics/infernal/cmsearch/main.nf

```diff
@@ -0,0 +1,66 @@
+
+process INFERNAL_CMSEARCH {
+
+    tag "$meta.id"
+
+    label 'process_high'
+
+    conda "bioconda::infernal=1.1.4"
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/infernal:1.1.4--pl5321hec16e2b_1':
+        'biocontainers/infernal:1.1.4--pl5321hec16e2b_1' }"
+
+    input:
+    tuple val(meta), path(seqdb)
+    path covariance_model_database
+
+    output:
+    tuple val(meta), path("*.cmsearch_matches.tbl.gz"), emit: cmsearch_tbl
+    path "versions.yml"                            , emit: versions
+
+    when:
+    task.ext.when == null || task.ext.when
+
+    script:
+    def args = task.ext.args ?: ''
+    def prefix = task.ext.prefix ?: "${meta.id}"
+
+    def is_compressed = seqdb.name.endsWith(".gz")
+    def seqdb_name = seqdb.name.replace(".gz", "")
+    """
+    if [ "$is_compressed" == "true" ]; then
+        gzip -c -d $seqdb > $seqdb_name
+    fi
+
+    cmsearch \\
+        --cpu $task.cpus \\
+        --noali \\
```

Comment:
They should be, I was fixing it but then I had to put on hold changes to this module and cmsearch-deoverlap modules because of this bug with cmsearch-deoverlap: https://github.com/nawrockie/cmsearch_tblout_deoverlap/issues/3

Sadly the developer has not replied since February.. might need to fix it ourselves and push it into a new docker container 😢